tests/tests_integration/test_api/test_transformations/test_transformations.py

```diff
@@ -26,11 +27,27 @@ def new_datasets(cognite_client):
 @pytest.fixture
 def new_transformation(cognite_client, new_datasets):
```

Comment:
can you also create a transformation without credentials and then update it with credentials and check if credentials exist?